### ISSUE TITLE
Link Checker Workflow is failing

### ISSUE THREAD

**Issue by A-Harby** [threefoldtech/info_grid#389]
https://github.com/threefoldtech/info_grid/actions/runs/7684813476

**Comment by A-Harby** [comment]
https://github.com/threefoldtech/info_grid/actions/runs/7685027723
https://github.com/threefoldtech/info_grid/actions/runs/7685318977

**Comment by Mik-TF** [comment]
Yes good catch. It means some 404 html errors are there. Will be able to fix them soon.

**Comment by Mik-TF** [comment]
Fixed when #390 is merged!

**Comment by A-Harby** [comment]
https://github.com/threefoldtech/info_grid/actions/runs/7694153090/job/20964385673 merged but still faling

**Comment by Mik-TF** [comment]
OK thanks for the info. I will fix it in a new PR asap.

Now development-split branch is available (it was used on another PR before), This means I can make changes on dev-split and it runs automatically the workflow, thus the link checker, so I can make sure the tests are properly completed before doing a PR for dev (and master).
